app/App/AccountSelector/AccountController/index.js

```diff
@@ -0,0 +1,383 @@
+      // Not currently selected, but another signer is
+      style.opacity = 0
+      style.pointerEvents = 'none'
+      style.transition = '300ms cubic-bezier(.82,0,.12,1) all'
+      if (this.store('selected.open')) {
+        // Not open, but another signer is
+        style.transform = this.props.index > this.store('selected.position.initial.index') ? 'translate(0px, 100px)' : 'translate(0px, -20px)'
+        style.opacity = 0
+        style.pointerEvents = 'none'
+      } else {
+        // style.transition = '400ms linear all'
+        style.transform = 'translate(0px, 0px)'
+        // style.transitionDelay = '400ms'
+        style.opacity = 1
+      }
+    } else {
+      if (this.store('view.addAccount')) {
+        style.opacity = 0
+        style.pointerEvents = 'none'
+      } else {
+        style.transition = '1.48s cubic-bezier(.82,0,.12,1) all'
+        style.transitionDelay = '0s'
+      }
+    }
+
+    const account = this.store('main.accounts', this.props.id)
+    let signer
+
+    if (account.signer) {
+      signer = this.store('main.signers', account.signer)
+    } else if (account.smart)  {
+      const actingSigner = this.store('main.accounts', account.smart.actor, 'signer')
+      if (actingSigner) signer = this.store('main.signers', actingSigner)
+    }
+
+    // const { data } = this.store('panel.nav')[0] || {}
+    // if (data && data.aux && data.aux.height) {
+    //   style.height = style.height - data.aux.height
+    // }
+
+    return (
+      <div 
+        className='signerWrap' 
+        style={current ? { height: initial.height + 'px' } : {}} 
+        onMouseDown={() => this.closeAccounts()}
+      >
+        <div 
+          className={signerClass}
+          style={style}
+          ref={ref => { 
+            if (ref) this.signer = ref 
+          }
+        }>
+          <div 
+              className={this.props.active ? 'signerTop signerTopActive' : 'signerTop'} 
+              style={open ? { boxShadow: '0px 4px 8px rgba(0, 0, 0, 0)' } : {}}
+              // onMouseEnter={() => this.setState({ openHover: true })} 
+              // onMouseLeave={() => this.setState({ openHover: false })}
+            >
+              {!this.state.addressHover ? (
+                <>
+                  {this.renderSignerIndicator()}
+                  {this.renderType()}
+                  {/* <div className='accountGrabber' style={open ? { opacity: 0, pointerEvents: 'none' } : {}}>
+                    {svg.grab(35)}
+                  </div> */}
+                  <div className='signerSelect' onClick={this.typeClick.bind(this)}>
+                    <div className='signerSelectIconWrap'>
+                      <div className='signerSelectIcon' style={{ transform: open ? 'rotate(180deg)' : 'rotate(0deg)' }}>
+                        {svg.chevron(26)}
+                      </div>
+                    </div>
+                  </div>
+                </>
+              ) : null}
+              {this.renderStatus()}
```

Comment:
This component probably the best candidate for splitting up, a lot going on and the status section pretty hard to understand